src/Compilers/CSharp/Portable/Syntax/SyntaxEquivalence.cs

```diff
@@ -82,6 +82,7 @@ private static bool AreTokensEquivalent(GreenNode? before, GreenNode? after, Fun
                 case SyntaxKind.NumericLiteralToken:
                 case SyntaxKind.CharacterLiteralToken:
                 case SyntaxKind.StringLiteralToken:
+                case SyntaxKind.UTF8StringLiteralToken:
```

Comment:
>SyntaxKind.UTF8StringLiteralToken [](http://example.com/codeflow?start=21&length=33)

Are we testing this case?

Reply:
> Are we testing this case?

Not explicitly. I'll make note in test plan